_data/projects/to-Chat.yml

```diff
@@ -0,0 +1,14 @@
+name: to-Chat (2C)
+desc: A chat box that lets you create group sessions for conversation, without the hassle of logging in.
+site: https://github.com/SamarthRanjanRay/2C
+tags:
+- *nodejs*
```

Comment:
I believe these `*` are unnecessary, and are causing issues with the parser. Can you drop them from the file?